magento/magento2-login-as-customer
# magento2-login-as-customer/issues/29: Destroy impersonated customer sessions on admin logout.

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. https://github.com/magento/magento2-login-as-customer/issues/29: Destroy impersonated customer sessions on admin logout.

### Manual testing scenarios (*)

1. See issue.
2. Wait for admin session to expire and make sure correcsponding login-as-customer session is destroyed.

### Questions or comments


### Contribution checklist (*)
